### PR TITLE
Updating troubleshooting docs

### PR DESCRIPTION
**Pull request by yannisle**
Hologram image color or brightness does not look right
Holograms look unstable
Lights to indicate problems

### CODE REVIEW

**Comment by yannisle** [comment]
Edits to troubleshooting docs

File: devices/hololens/hololens-troubleshooting.md [hunk 1]
```diff
@@ -33,14 +33,22 @@ If your HoloLens becomes frozen or unresponsive:
...
 If these steps don't work, you can try [recovering your device](hololens-recovery.md).
 
-## Holograms don't look good
+## Hologram image color or brightness does not look right
+For HoloLens 2, take the following steps to ensure the highest visual quality of holograms presented in displays:
+
+- **Increase brightness of the display:** Holograms look best when the display is at its brightest level.
+- **Bring visor closer to your eyes:** Swing the visor down to the closest position to your eyes.
+- **Shift visor down:** Try moving the brow pad on your forehead down, which will result in the visor moving down closer to your nose.
+- **Run eye calibration:** The display uses your IPD and eye gaze to optimize images on the display. If you don't run eye calibration, the image quality may be made worse.
```
[the remark this answers is not in the document]
Comment (scooley) [reply devices/hololens/hololens-troubleshooting.md]: There are a number of environmental factors that may also change how your holograms look including light level.  Read more about environmental considerations [here](https://docs.microsoft.com/hololens/hololens-environment-considerations) if the steps above don't work.

`[here](hololens-environment-considerations.md)`

File: devices/hololens/hololens-troubleshooting.md [hunk 2]
```diff
@@ -52,6 +60,14 @@ If your environment is too dark, HoloLens might not see your hand, so make sure
...
 If your visor has fingerprints or smudges, use the microfiber cleaning cloth that came with the HoloLens to clean your visor gently.
 
+## Lights to indicate problems
+
+| When you do this | The lights do this | It means this |
+| - | - | - |
+| You press the Power button. | One light flashes five times, then turns off. | The HoloLens battery is critically low. Charge your HoloLens. |
+| You press the Power button. | All five lights flash five times, then turn off. |  HoloLens cannot start correctly and is in an error state. [Reinstall the operating system](hololens-recovery.md) to recover your device. |
+| You press the Power button. | The 1st, 3rd, and 5th lights flash together continually. |  HoloLens may have a hardware failure. To be sure, [reinstall the OS](hololens-recovery.md#hololens-2), and try again. After reinstalling the OS, if the light-flash pattern persists, contact [support](https://support.microsoft.com/en-us/supportforbusiness/productselection?sapid=3ec35c62-022f-466b-3a1e-dbbb7b9a55fb). |
+
```
Comment (scooley) [inline devices/hololens/hololens-troubleshooting.md]: Here is the light guide for [HoloLens (1st gen)](https://docs.microsoft.com/hololens/hololens1-setup#hololens-indicator-lights) and [HoloLens 2](https://docs.microsoft.com/hololens/hololens2-setup#hololens-behavior-reference).

`[HoloLens (1st gen)](hololens1-setup.md#hololens-indicator-lights)`
`[HoloLens 2](hololens2-setup.md#hololens-behavior-reference)`

**Comment by e0i** [comment]
@yannisle Please let us know when you are able to follow up with revision suggestions.Thanks.

**Comment by yannisle** [comment]
Reviewed revisions.

File: devices/hololens/hololens-troubleshooting.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (illfated) [reply devices/hololens/hololens-troubleshooting.md]: This link has not been added. Sorry, I forgot to check the previous reviews.

**Comment by illfated** [comment]
> Reviewed revisions.

Maybe you would like specific suggestions within the source text, to be merged by using the button **[Commit suggestion]** ?

**Comment by e0i** [comment]
@yannisle Good to go once the merge conflicts have been resolved. Thank you.

File: devices/hololens/hololens-troubleshooting.md [hunk 3]
```diff
@@ -52,6 +60,14 @@ If your environment is too dark, HoloLens might not see your hand, so make sure
 
 If your visor has fingerprints or smudges, use the microfiber cleaning cloth that came with the HoloLens to clean your visor gently.
 
+## Lights to indicate problems
+
+| When you do this | The lights do this | It means this |
+| - | - | - |
+| You press the Power button. | One light flashes five times, then turns off. | The HoloLens battery is critically low. Charge your HoloLens. |
+| You press the Power button. | All five lights flash five times, then turn off. |  HoloLens cannot start correctly and is in an error state. [Reinstall the operating system](hololens-recovery.md) to recover your device. |
+| You press the Power button. | The 1st, 3rd, and 5th lights flash together continually. |  HoloLens may have a hardware failure. To be sure, [reinstall the OS](hololens-recovery.md#hololens-2), and try again. After reinstalling the OS, if the light-flash pattern persists, contact [support](https://support.microsoft.com/supportforbusiness/productselection?sapid=3ec35c62-022f-466b-3a1e-dbbb7b9a55fb). |
```
Comment (JohanFreelancer9) [inline devices/hololens/hololens-troubleshooting.md:69]:
```suggestion
| You press the Power button. | The first, third, and fifth lights flash together continually. |  HoloLens may have a hardware failure. To be sure, [reinstall the OS](hololens-recovery.md#hololens-2), and try again. After reinstalling the OS, if the light-flash pattern persists, contact [support](https://support.microsoft.com/supportforbusiness/productselection?sapid=3ec35c62-022f-466b-3a1e-dbbb7b9a55fb). |
```

Comment (JohanFreelancer9) [reply devices/hololens/hololens-troubleshooting.md:69]:
```suggestion
| You press the Power button. | The first, third, and fifth lights flash together continually. |  HoloLens may have a hardware failure. To be sure, [reinstall the OS](hololens-recovery.md#hololens-2), and try again. After reinstalling the OS, if the light-flash pattern persists, contact [support](https://support.microsoft.com/supportforbusiness/productselection?sapid=3ec35c62-022f-466b-3a1e-dbbb7b9a55fb). |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edits to align with Microsoft Style Guide.

**Comment by e0i** [comment]
@yannisle Please feel free to re-open this PR by resolving the merge conflicts and applying the copy editing suggestions. The content is ready to merge once that’s been done. Thank you for the feedback and for your patience with the process.

**Pull request closed without merging**